src/swig_python/swig.i

```diff
@@ -354,9 +324,12 @@ static void destroy_words(PyObject *obj) { (void)obj; }
 %rename("bip32_key_init") bip32_key_init_alloc;
 %rename("bip32_key_unserialize") bip32_key_unserialize_alloc;
 %rename("bip32_key_with_tweak_from_parent_path") bip32_key_with_tweak_from_parent_path_alloc;
-%rename("keypath_map_init") wally_keypath_map_init_alloc;
-%rename("partial_sigs_map_init") wally_partial_sigs_map_init_alloc;
+%rename("psbt_clone") wally_psbt_clone_alloc;
 %rename("psbt_get_global_tx") wally_psbt_get_global_tx_alloc;
+%rename("psbt_get_input_final_witness") wally_psbt_get_input_final_witness_alloc;
+%rename("psbt_get_input_utxo") wally_psbt_get_input_utxo_alloc;
+%rename("psbt_get_input_pegin_tx") wally_psbt_get_input_pegin_tx_alloc;
+%rename("psbt_get_input_witness_utxo") wally_psbt_get_input_witness_utxo_alloc;
 %rename("psbt_init") wally_psbt_init_alloc;
```

Comment:
We should the same for `wally_psbt_elements_init_alloc`

Reply:
Done, thanks!